resources/turso/src/lib.rs

```
@@ -120,20 +123,23 @@ impl ResourceBuilder<Client> for Turso {
                             conn_url: Url::parse(&conn_url).map_err(Error::UrlParseError)?,
                             // Nullify the token since we're using a file as database.
                             token: None,
+                            environment: md.env,
                         })
                     }
                 }
             }
         }
     }
 
-    async fn build(config: &Self::Output) -> Result<Client, shuttle_service::Error> {
-        let client = Client::from_config(Config {
-            url: config.conn_url.clone(),
-            auth_token: config.token.clone(),
-        })
-        .await?;
-        Ok(client)
+    async fn build(config: &Self::Output) -> Result<Database, shuttle_service::Error> {
+        let database = match config.environment {
+            Environment::Deployment => Database::open_remote(
+                config.conn_url.to_string(),
+                config.token.clone().unwrap().to_string(),
```

Should we rather return an error here if the token is missing?